execution_engine2.spec

```diff
@@ -22,20 +35,26 @@ module execution_engine2 {
         mapping<string, string> config;
         string git_commit;
     } Status;
+
+    /* Returns the service configuration, including URL endpoints and timeouts */
```

Comment:
Should we add
```
        public_keys = ['external-url', 'kbase-endpoint', 'workspace-url', 'catalog-url',
                       'shock-url', 'handle-url', 'auth-service-url', 'auth-service-url-v2',
                       'auth-service-url-allow-insecure',
                       'scratch', 'executable', 'docker_timeout',
                       'intialdir', 'transfer_input_files']
```

Reply:
Sure. And can we fix the typo in `intialdir`? Maybe not in this PR, though, seems a touch out of scope.

Reply:
Done, and respelled as "initial_dir" - that's what's used in the actual config file. I'll make a separate PR with a code update.